web/components/share-embed-button.tsx

```diff
@@ -9,13 +9,23 @@ import { DOMAIN } from 'common/envs/constants'
 import { copyToClipboard } from 'web/lib/util/copy'
 import { track } from 'web/lib/service/analytics'
 
-export function embedCode(contract: Contract) {
+export function embedContractCode(contract: Contract) {
   const title = contract.question
   const src = `https://${DOMAIN}/embed${contractPath(contract)}`
+  return embedCode(src, title)
+}
 
-  return `<iframe width="560" height="405" src="${src}" title="${title}" frameborder="0"></iframe>`
+export function embedContractGridCode(contracts: Contract[]) {
+  //TODO: Change before shipping
+  const src = `http://localhost:3000/embed/grid/${contracts
+    .map((c) => c.slug)
+    .join('/')}`
+  return embedCode(src, 'Grid of contracts')
 }
 
+const embedCode = (src: string, title: string) =>
+  `<iframe width="560" height="405" src="${src}" title="${title}" frameborder="0"></iframe>`
```

Comment:
@akrolsmir  @sinclairity  How can I make the iframe height cover the content? As far as I can tell those width and height attributes are not doing anything:

<img width="1726" alt="image" src="https://user-images.githubusercontent.com/6242616/187687172-ca2b2857-cd65-4be4-b443-af0099b72d79.png">

Reply:
I think it's actually controlled by the `padding-bottom: 20rem;` here. No idea why, but increasing it to 40rem does increase the iframe height, probably because the iframe itself is using `h-full` and ignoring the other params

<img width="1287" alt="image" src="https://user-images.githubusercontent.com/856034/187794977-77e5a55d-a96d-474b-a1a7-6096db5564ae.png">

Reply:
It's in https://github.com/manifoldmarkets/manifold/blob/26aba26da538fa978f1b0f205f5e09647709e362/common/util/tiptap-iframe.ts#L38

Simplest fix would be to add a param to dynamically control the iframe height, maybe?

Not really sure why the CSS works this way fwiw, I basically just copied the https://tiptap.dev/experiments/embeds example and hacked over it, very open to rearranging